Classes/Package.php

```diff
@@ -21,23 +23,52 @@ public function boot(Bootstrap $bootstrap)
         $dispatcher = $bootstrap->getSignalSlotDispatcher();
 
         $dispatcher->connect(Node::class, 'nodePropertyChanged', function (NodeInterface $node, $propertyName, $_, $newValue) use ($bootstrap) {
```

Comment:
not new but it should be fixed IMO: `$bootstrap` is not used in the closure
```suggestion
        $dispatcher->connect(Node::class, 'nodePropertyChanged', function (NodeInterface $node, $propertyName, $_, $newValue) {
```